src/state/settings.ts

```diff
@@ -17,7 +17,7 @@ const asyncStorageEffect =
 
 export const envState = atom<keyof AccessKeys>({
   key: 'envState',
-  default: 'sandbox',
+  default: asyncStorage.getItem('env').then((data) => (data ? JSON.parse(data) : 'sandbox')),
```

Comment:
Do we need to use the default value here? asyncStorageEffect already gets default value if I understand correctly